charts/telegraf/templates/deployment.yaml

```diff
@@ -41,6 +41,11 @@ spec:
         {{- end }}
         env:
 {{ toYaml .Values.env | indent 8 }}
+{{- if .Values.envFromSecret }}
+        envFrom:
+          - secretRef:
+              name: {{ tpl .Values.envFromSecret . }}
```

Comment:
This should be `.Values.envFromSecret` rather than using `tpl`, as the value in the values file won't be a template itself.

Reply:
@rawkode you're totally correct, I somewhat overthought this 🙂 Now I see we don't have any secret references in `values.yaml`